README.md

```diff
@@ -68,6 +68,19 @@ be highlighted.
 
     pip install Pygments
 
+If you don't have `pip` installed yet, you'll first need to install it.
+
+On Mac OS you can do so by installing Python with homebrew
+
+    brew install python
+
+We use HTMLProofer to check whether hyperlinks are correct. To run HTMLProofer on your local version, run `make htmlproofer` in the directory
+where this project is located on your machine.
+
+If this doesn't work, you may need to install Ruby first:
```

Comment:
Perhaps: "If the following does not work"

Reply:
"this" actually refers to `make htmlproofer` (I didn't have ruby yet...)